Jenkinsfile

```diff
@@ -69,33 +69,35 @@ fabric8UITemplate{
 
 // deploy a snapshot fabric8-ui pod and notify pull request of details
 if (ciDeploy){
-   def prj = 'fabric8-ui-'+ env.BRANCH_NAME
-   prj = prj.toLowerCase()
-   def route
-   deployOpenShiftNode(openshiftConfigSecretName: 'fabric8-intcluster-config'){
-       stage("deploy ${prj}"){
-           route = deployOpenShiftSnapshot{
-               mavenRepo = 'http://central.maven.org/maven2/io/fabric8/online/apps/fabric8-ui'
-               githubRepo = 'fabric8-ui'
-               originalImageName = 'registry.devshift.net/fabric8-ui/fabric8-ui'
-               newImageName = imageName
-               openShiftProject = prj
-               githubProject = project
-           }
-       }
-       stage('notify'){
-           def changeAuthor = env.CHANGE_AUTHOR
-           if (!changeAuthor){
-               error "no commit author found so cannot comment on PR"
-           }
-           def pr = env.CHANGE_ID
-           if (!pr){
-               error "no pull request number found so cannot comment on PR"
-           }
-           def message = "@${changeAuthor} ${imageName} is deployed and available for testing at https://${route}"
-           container('clients'){
-               flow.addCommentToPullRequest(message, pr, project)
-           }
-       }
-   }
+    def prj = 'fabric8-ui-'+ env.BRANCH_NAME
+    prj = prj.toLowerCase()
+    def route
+    timeout(time: 10, unit: 'MINUTES') {
```

Comment:
@pranavgore09 Is 10 minutes timeout sufficient?

Reply:
sure! I can set the smaller value by observing avg time from builds. I think 5m will be sufficient too

Reply:
:+1: